app/src/org/commcare/android/logging/ReportingUtils.java

```diff
@@ -91,4 +94,14 @@ public static String getVersion() {
         }
     }
 
+    public static String getCommCareVersionString() {
```

Comment:
is this doing anything different from https://github.com/dimagi/commcare-android/blob/bbabdbc72fb8ecd04d8582f432f6634d3c7e1b6a/app/src/org/commcare/AppUtils.java#L140-L140 ?